integration_tests/integration/bookedtoday/arrivals/matchingRecords/possibleRecordsFound.spec.ts

```diff
@@ -0,0 +1,91 @@
+import ChoosePrisonerPage from '../../../../pages/bookedtoday/choosePrisoner'
+import PossibleRecordsFoundPage from '../../../../pages/bookedtoday/arrivals/matchingRecords/possibleRecordsFound'
+import SearchForExistingPage from '../../../../pages/bookedtoday/arrivals/searchforexisting/searchForExisting'
+import ImprisonmentStatus from '../../../../pages/bookedtoday/arrivals/imprisonmentStatus'
+
+import Page from '../../../../pages/page'
+import Role from '../../../../../server/authentication/role'
+import expectedArrivals from '../../../../mockApis/responses/expectedArrivals'
+
+const matchedRecords = [
+  {
+    firstName: 'Sam',
+    lastName: 'Smith',
+    dateOfBirth: '1970-02-01',
+    prisonNumber: 'G0014GM',
+    pncNumber: '01/1111A',
+    croNumber: '01/0000A',
+    sex: 'MALE',
+  },
+  {
+    firstName: 'Sammy',
+    lastName: 'Smith',
+    dateOfBirth: '1970-02-01',
+    sex: 'MALE',
+  },
+]
+
+context('Possible existing records', () => {
+  beforeEach(() => {
+    cy.task('reset')
+    cy.task('stubSignIn', Role.PRISON_RECEPTION)
+    cy.task('stubPrison', 'MDI')
+    cy.task('stubAuthUser')
+    cy.task('stubUserCaseLoads')
+    cy.task('stubTransfers', { caseLoadId: 'MDI', transfers: [] })
+    cy.task('stubMissingPrisonerImage')
+    const arrival = expectedArrivals.arrival({
+      fromLocationType: 'COURT',
+      isCurrentPrisoner: false,
+      dateOfBirth: '1972-11-21',
+      pncNumber: null,
+      prisonNumber: null,
+    })
+    cy.task('stubExpectedArrivals', { caseLoadId: 'MDI', arrivals: [arrival] })
+    cy.task('stubExpectedArrival', arrival)
+    cy.task('stubMatchedRecords', matchedRecords)
+    cy.task('stubPrisonerDetails', matchedRecords[0])
+    cy.task('stubImprisonmentStatus')
+    cy.signIn()
+
+    const choosePrisonerPage = ChoosePrisonerPage.goTo()
+    choosePrisonerPage.arrivalFrom('COURT')(1).confirm().click()
+  })
+
+  it('should display page contents', () => {
+    const possibleRecordsFoundPage = PossibleRecordsFoundPage.goTo('11111-11111')
+    possibleRecordsFoundPage.perElement('prisoner-name').should('contain', 'Bob Smith')
+    possibleRecordsFoundPage.perElement('dob').should('contain', '21 November 1972')
+    possibleRecordsFoundPage.matchingElement('1-prisoner-name').should('contain', 'Sam Smith')
+    possibleRecordsFoundPage.matchingElement('1-dob').should('contain', '1 February 1970')
```

Comment:
Worth thinking upfront what an ideal interface to work with would be and then designing the code to implement that interface. 

There's obviously lots of possible version of "ideal", one such possibility: 
```
const firstMatch = possibleRecordsFoundPage.match(1)
firstMatch.field('prisoner-name').should('contain', 'Bob Smith')
firstMatch.field('dob').should('contain', '21 November 1972')
...
firstMatch.select().click()
```

It's not great passing untyped strings.. properties would be easy to move to - if you really wanted to strings then it would be better to type it to a union of possible fields